app/views/profile/show.html.erb

```diff
@@ -2,35 +2,37 @@
   <%= render 'users/header' %>
 
   <!--main content-->
-  <% unless @user.profile.title.blank? %>
-    <h3 class="heading"><%=h @user.profile.title %></h3>
+  <% if @profile.title.present? %>
+    <h3 class="heading"><%=h @profile.title %></h3>
   <% end %>
 
   <div class="wrapper">
     <dl class="meta">
-      <dt class="pseuds"><%= link_to ts("My pseuds:"), user_pseuds_path(@user) %></dt>
-      <dd class="pseuds"><%= print_pseud_list(@user.pseuds) %></dd>
+      <dt class="pseuds">
+        <%= link_to all_pseuds_link(@profile), user_pseuds_path(@user) %>
+      </dt>
+      <dd class="pseuds"><%= print_pseud_list(@profile.pseuds) %></dd>
       <dt><%= ts("I joined on:") %></dt>
-      <dd><%= ts("%{date}", :date => l(@user.created_at.to_date)) %></dd>
-      <% if @user.profile.location? %>
+      <dd><%= ts("%{date}", :date => l(@profile.created_at)) %></dd>
```

Comment:
`l()` should be sufficient to localize the date, shouldn't it? Is the `ts("%{date}")` part necessary?

(Not really in scope, but I noticed it while I was scrolling through the code.)

Reply:
Fixed in my PR to Cristina